ghost/admin/app/components/posts/feedback-events-chart.js

```diff
@@ -1,22 +1,35 @@
 import Component from '@glimmer/component';
+import {action} from '@ember/object';
 import {inject as service} from '@ember/service';
-
-const CHART_COLORS = [
-    '#F080B2',
-    '#8452f633'
-];
-
-const linksClass = ['gh-post-activity-chart-positive-feedback', 'gh-post-activity-chart-negative-feedback'];
+import {tracked} from '@glimmer/tracking';
 
 export default class FeedbackEventsChart extends Component {
     @service feature;
+    @tracked tooltipData = {};
+    tooltipNode = null;
+
+    @action
+    onTooltipInsert(node) {
+        this.tooltipNode = node;
+    }
+
+    @action
+    onMouseleave() {
+        this.tooltipNode.style.opacity = '0';
+        this.tooltipNode.style.position = 'fixed';
+        this.tooltipNode.style.left = '2000';
```

Comment:
Some remarks here:
- Can't we only set opacity to 0 and leave out the rest? (I think setting left to '2000' is invalid because it is lacking a unit). I think displaying the tooltip at some position outside the window can cause scrolling issues on some devices.
- Don't we need to also disable pointer events or set display to none to avoid interaction when it is hidden? Setting opacity to 0 alone doesn't disable interaction.

Reply:
Changed to `display: 'none'`